libraries/bot-applicationinsights/src/main/java/com/microsoft/bot/applicationinsights/BotTelemetryClientImpl.java

```diff
@@ -53,9 +51,16 @@ public BotTelemetryClientImpl(TelemetryClient withTelemetryClient)
      * @param properties Named string values you can use to classify and search for this availability telemetry.
      * @param metrics Additional values associated with this availability telemetry.
      */
+    @SuppressWarnings("checkstyle:ParameterNumber")
```

Comment:
Is this ok?

Reply:
The checkstyle complains about the amount of parameter in the constructor, we had to add that just like we did in [QnAMaker class](https://github.com/microsoft/botbuilder-java/blob/main/libraries/bot-ai-qna/src/main/java/com/microsoft/bot/ai/qna/dialogs/QnAMakerDialog.java#L489)

Reply:
Ack